StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsSheetConfiguration.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  SavedPaymentMethodsSheetConfiguration.swift
+//  StripePaymentSheet
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) @_spi(PrivateBetaSavedPaymentMethodsSheet) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+extension SavedPaymentMethodsSheet {
+
+    public struct Configuration {
+        public typealias CreateSetupIntentHandlerCallback = ((@escaping (String?) -> Void) -> Void)
+
+        private var styleRawValue: Int = 0  // SheetStyle.automatic.rawValue
+        /// The color styling to use for PaymentSheet UI
+        /// Default value is SheetStyle.automatic
+        /// @see SheetStyle
+        @available(iOS 13.0, *)
```

Comment:
We can remove `@available` (and the below hacky comment and private var), as we require iOS 13